src/translations/locale/locale.en-US.json

```diff
@@ -0,0 +1,4 @@
+{
+  "validation.email.invalid": "Email is not correct",
+  "validation.required": "Field is required"
+}
```

Comment:
Yeah, indeed, I would want to just write `value="some.id"` and it should know how to properly get the string based on the id and the language I am using. What do you mean by 'doesnt work with the translation messages'?